01-html/04-nytimes.html

```diff
@@ -1,3 +1,1120 @@
+          <li><a href="http://www.nytimes.com/video/style">Style</a></li>
+          <li><a href="http://www.nytimes.com/video/t-magazine">T Magazine</a></li>
+          <li><a href="http://www.nytimes.com/video/health">Health</a></li>
+          <li><a href="http://www.nytimes.com/video/dining-and-wine">Food</a></li>
+          <li><a href="http://www.nytimes.com/video/travel">Travel</a></li>
+          <li><a href="http://www.nytimes.com/video/sports">Sports</a></li>
+          <li><a href="http://www.nytimes.com/video/real-estate">Real Estate</a></li>
+          <li><a href="http://www.nytimes.com/video/science">Science</a></li>
+        </ul>
+        <li><a href="http://www.nytimes.com/upshot/">The Upshot</a></li>
+        <li><a href="">Conferences</a></li>
+        <ul>
+          <li><a href="http://nytlive.nytimes.com/womenintheworld/">Women in the World</a></li>
+          <li><a href="http://inytenergyfortomorrow.com/">Energy for Tomorrow</a></li>
+          <li><a href="http://www.newworksummit.com/">New Work Summit</a></li>
+          <li><a href="http://luxurybeyondproduct.com/">The Luxury Conference</a></li>
+          <li><a href="http://artfortomorrow.com/">Art for Tomorrow</a></li>
+        </ul>
+        <li><a href="http://www.nytimes.com/crosswords/index.html?page=home&_r=0">Crossword</a></li>
+        <li><a href="http://www.nytimes.com/section/insider">Times Insider</a></li>
+        <li><a href="http://www.nytimes.com/pages/multimedia/index.html">Multimedia</a></li>
+        <li><a href="http://lens.blogs.nytimes.com/">Photography</a></li>
+        <li><a href="https://www.nytimes.com/store/">NYT Store</a></li>
+        <li><a href="http://www.nytwineclub.com/">NYT Wine Club</a></li>
+        <li><a href="https://nytedu.com/">nytEducation</a></li>
+        <li><a href="">More</a></li>
+        <ul>
+          <li><a href="http://www.nytimes.com/times-journeys/">Times Journeys</a></li>
+          <li><a href="http://www.nytimes.com/subscriptions/">Subscribe</a></li>
+          <li><a href="https://myaccount.nytimes.com/">Manage Account</a></li>
+          <li><a href="http://www.nytimes.com/pages/todayspaper/index.html">Today's Paper</a></li>
+          <li><a href="http://www.nytimes.com/interactive/blogs/directory.html">Blogs</a></li>
+          <li><a href="http://www.nytimes.com/pages/topics/">Times Topics</a></li>
+          <li><a href="http://www.nytimes.com/marketing/tools-and-services/">Tools & Services</a></li>
+          <li><a href="http://www.nytimes.com/section/jobs">Jobs</a></li>
+          <li><a href="http://www.nytimes.com/ref/classifieds/">Classifieds</a></li>
+          <li><a href="http://www.nytimes.com/pages/corrections/index.html">Corrections</a></li>
+        </ul>
+    </nav>
+    <div id="search" class="inline-block">
+      <img src="http://uploads.webflow.com/55cf187a8cacc9ab28d524b9/55d243680a3ebde1455ffab4_transparency.png" height="30" alt="search-icon">SEARCH</div>
```

Comment:
Generally, unless an image is content, you ought to favor using css background images instead of `img` tags. Open a NYTimes article in Safari, enter Reader Mode, and take a look at the images it displays - those are the content.